stories/components/theme.tsx

```diff
@@ -1,49 +1,58 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at http://mozilla.org/MPL/2.0/
 
 import * as React from 'react'
-// Storybook requires
-import { storiesOf, setAddon } from '@storybook/react'
-// import { withKnobs, boolean, select } from '@storybook/addon-knobs'
-// @ts-ignore
-import chaptersAddon from 'react-storybook-addon-chapters'
-
-// Components
+import { storiesOf } from '@storybook/react'
 import styled from '../../src/theme'
 import paletteColors from '../../src/theme/palette'
 import themeBraveDefault from '../../src/theme/brave-default'
-import '../assets/fonts/muli.css'
 
-// wrappers
+const Palette = styled.div`
+  display: grid;
+  grid-gap: 8px;
+  grid-template-columns: repeat(10, 1fr);
+  @media only screen and (max-width: 1300px) {
+    grid-template-columns: repeat(5, 1fr);
+  }
+` as any
```

Comment:
this is ok for now as other components here use it but let's keep track of `any` next time we revist this file and remove all